src/identity/utils/convertChainIdToCoinType.ts

```diff
@@ -0,0 +1,14 @@
+import { mainnet } from 'viem/chains';
+
+/**
+ * Convert an chainId to a coinType hex for reverse chain resolution
+ */
+export const convertChainIdToCoinType = (chainId: number): string => {
```

Comment:
what's a coin? :D

Reply:
I think it's fair to link somehere this https://docs.ens.domains/ensip/11 for future reference.